<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>375</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="clientOptions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="clientLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>640</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Java Client Options</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="serverLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>640</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Java Server Options</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="serverOptions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>145</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset to Default</string>
   </property>
  </widget>
  <widget class="QLabel" name="configUrlLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>185</y>
     <width>640</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Config URLs (select one for XMage and Java downloads)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="configUrlList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>640</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addUrlButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeUrlButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>340</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
